### ISSUE TITLE
Detect 'sort by' in query and use to set table column sort

### ISSUE THREAD

**Issue by ijdickinson** [epimorphics/qonsole#7]
From Andy:

> The qonsole HTML table sorted by the first column by default.  But if the query has got an ORDER BY in it, this aspect is lost.
> One possibility: if the query contains the string "ORDER BY" then take the natural order of the query with no column ordering.

**Comment by bwmcbride** [comment]
Taking the natural order of the query would be good.

**Comment by ijdickinson** [comment]
So it turns out that I had already fixed this problem, but not merged it into `master`. I've now done that merge, but in so doing I realise that the code is a bit crufty, not least from the tools perspective. We no longer tend to use bower to manage dependencies, for example, and now use webpack or browserify to flatten the dependencies down into single files. 

Tl;dr: you should be able to use the `0.2.0` release and see the sort order working correctly, but really I need to start working on updating the codebase to `0.3.0`.

**Comment by paperclipmonkey** [comment]
Marking as closed
